### ISSUE TITLE
How do you provide the standalone? Do you compile it yourself? If so can you document the workflow (for posterity) ? 

### ISSUE THREAD

**Issue by luzpaz** [HaasNL2/Cycles-stand-alone-windows-build#1]

**Comment by MisterMakerNL** [comment]
Yes, I compile it myself instructions can be found here:  https://github.com/blender/cycles/blob/main/BUILDING.md
I did even build it again more recently but non of the updates had any impact on the results, I even had some issues so I decided to keep the older version here.
Maybe I have to fork it and build releases but I have no clue atm how to do this.
